ext/include/opentelemetry/ext/http/client/http_client.h

```diff
@@ -37,6 +39,17 @@
   session->FinishSession() // optionally in the end
   ...shutdown
   sessionManager.FinishAllSessions()
+
+Sync Request:
+
+  SessionMamager sessionManager;
+  auto session = sessionManager.createSession("localhost", 8000);
+  auto request = session->CreateRequest();
+  request->AddHeader(..);
+  SessionState session_state;
```

Comment:
So this is not a `session` state - this is given `request` state?

Reply:
Correct.